clients/cloud/csharp/Program.cs

```diff
@@ -136,26 +152,31 @@ static void Consume(string topic, ClientConfig config)
                 {
                     while (true)
                     {
-                        var cr = consumer.Consume();
+                        var cr = consumer.Consume(cts.Token);
                         totalCount += JObject.Parse(cr.Value).Value<int>("count");
                         Console.WriteLine($"Consumed record with key {cr.Key} and value {cr.Value}, and updated total count to {totalCount}");
                     }
                 }
                 catch (OperationCanceledException)
                 {
                     // Ctrl-C was pressed.
-                    Console.WriteLine("closed");
+                }
+                finally
+                {
                     consumer.Close();
                 }
             }
         }
 
+        static void PrintUsage()
+        {
+            Console.WriteLine("usage: .. <produce|consume> topic configPath [certDir]");

```

Comment:
So, the typical command syntax is `<..>` for variable fields, thus:
`Usage: .. produce|consume <topic> <configPath> [<certDir>]`

Reply:
thanks for the education - didn't know that, wondered... (will change)